guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -34,9 +34,9 @@ Ensure that you understand the following terms:
 
 To clone {ProjectServer}, ensure that you have the following resources available:
 
-* A minimal install of {EL} 7 to become the target server.
-Do not install {EL} 7 software groups, or third-party applications.
-Ensure that your server complies with all the specifications of {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _Installing {ProjectServer}_.
+* A minimal install of {EL} 8 to become the target server.
+Do not install {EL} 8 software groups, or third-party applications.
+Ensure that your server complies with all the specifications of {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.
```

Comment:
Alternative:

```suggestion
Ensure that your server complies with all the required specifications.
For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.
```

I am unsure if my suggestions adds a benefit apart from consistency (which is a benefit in itself). Your call.